runtime/hydra-dx/src/lib.rs

```diff
@@ -145,8 +160,8 @@ pub fn native_version() -> NativeVersion {
 
 pub struct BaseFilter;
 
-impl Filter<Call> for BaseFilter {
-	fn filter(call: &Call) -> bool {
+impl Contains<Call> for BaseFilter {
+	fn contains(call: &Call) -> bool {
 		match call {
 			Call::Authorship(_)
```

Comment:
note that rest of the calls dont have to be listed in match and you can just match rest (_) as true